task/transcode.go

```diff
@@ -161,6 +161,13 @@ func TaskTranscode(tctx *TaskContext) (*data.TaskOutput, error) {
 	outFiles = append(outFiles, mp4muxer)
 	outBuffers = append(outBuffers, ws)
 	var transcoded [][]byte
+	contentResolution := ""
```

Comment:
Should check either here or in the client that the `contentResolution` value is not empty (or if we change it to width/height, that it is `> 0`). We shouldn't set the header with an empty value.

Even though we theoretically don't have asses without video tracks / resolution info right now, this could change in the future and it's good to have the code reliable to that.

Reply:
Do you know if the broadcaster would support a value like that? If so, that could be cool :)